datafusion/physical-plan/src/limit.rs

```diff
@@ -188,21 +188,11 @@ impl ExecutionPlan for GlobalLimitExec {
     fn statistics(&self) -> Result<Statistics> {
```

Comment:
Thanks for noticing the error and fixing it. I think the new logic here is quite reasonable.